MAUI/Calendar/views.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: Views in MAUI Calendar widget | Syncfusion
+description: Learn here about the Views feature of Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: Calendar
+documentation: ug
+---
+
+
+# Multiple Calendar Views in MAUI (SfCalendar)
+The `SfCalendar` widget has four Calendar views to display. It can be assigned to the widget by using the `View` property. `Month` view is initially rendered by Default. The current date will be displayed initially for all the Calendar views.
```

Comment:
The `SfCalendar` widget has four Calendar views to display. It can be assigned to the widget by using the `View` property. `Month` view is initially rendered by Default. The current date will be displayed initially for all the Calendar views.

Reply:
Content modified.